craft_parts/errors.py

```diff
@@ -52,15 +52,7 @@ def __init__(
         self.doc_slug = doc_slug
 
     def __str__(self) -> str:
-        components = [self.brief]
-
-        if self.details:
-            components.append(self.details)
-
-        if self.resolution:
-            components.append(self.resolution)
-
-        return "\n".join(components)
+        return self.brief
```

Comment:
I still don't understand how this error is ending up in craft-cli?

Reply:
I linked to it